installers/Windows/README.md

```diff
@@ -41,4 +41,56 @@ https://github.com/mu-editor/mu/blob/master/win_installer.py
 
 Files covered:
 
-installer.py
\ No newline at end of file
+installer.py
+
+## Step by Step: Creating a Spyder Windows Installer
+To create a Spyder Windows Installer you must follow the following steps.
+
+### Creating a Conda Environment 
+- Download and install Anaconda. You can find the latest version [here](https://www.anaconda.com/products/distribution). 
+- Open the Anaconda prompt and make sure that you have admin permissions.
+- Create the new enviroment with the following commands:
+bash
+$ conda create -n installer python=3.8.10
+$ conda activate installer
+
+The first command creates an enviroment called "installer" with Python version 3.8.10 and the second one, changes the current enviroment to the new one, "installer".'
+
+After changing to the new enviroment, install yarg with pip. 
+bash
+pip install yarg
+
+### Creating Enviroment Variables
+Before running any script, you must set some enviroment variables first. 
+The list below shows all the variables that must be created.
+-   ASSETS_URL=https://github.com/spyder-ide/windows-installer-assets/releases/download/0.0.7/assets.zip
+-   UNWANTED_PACKAGES=pip spyder-kernels python-slugify Rtree QDarkStyle PyNaCl
+-   SKIP_PACKAGES=bcrypt slugify
+-   ADD_PACKAGES=spyder_kernels blib2to3 _black_version blackd rtree qdarkstyle nacl
+-   PYNSIST_REQ=pynsist==2.7
+-   EXE_NAME=Spyder_64bit_full.exe
+
+For each variable on the list, type
+`set <variable=value>` in the anaconda prompt. 
+### Running the Installer Script
+Type the following line into the anaconda command prompt:
+bash
+python installers/Windows/installer.py 3.8.10 64 setup.py spyder.app.start:main spyder img_src/spyder.ico LICENSE.txt -ep installers/Windows/req-extras-pull-request.txt -s full -t installers/Windows/assets/nsist/spyder.nsi
+
+where:
+
+- `installers/Windows/installer.py` is the path to the installer script
+- `3.8.10` is the Python version 
+- `64` is the bitness of the installer
+- `setup.py`  is the path to the setup script
+- `spyder.app.start:main` is the entrypoint to execute the package
+- `spyder` is the name of the package
+- `img_src/spyder.ico` is the path to the Spyder installer icon.
+- `LICENSE.txt`  is the path to the license file
+- `installers/Windows/req-extras-pull-request.txt` is the path to the file with the requirements needed besides the dependencies of the main package.
+-  `installers/Windows/assets/nsist/spyder.nsi` is the path .nsi template for the installer
+
+Additionaly,
+- -ep means extra packages 
+- -s means suffix for the name of the generated executable (can be full or lite)
+- -t means template
```

Comment:
We could also add here that the `installer.py` has `--help` with the description of all the arguments available so (depending on your current working directory) you can run something like `python installers/Windows/installer.py --help`